src/OpenTelemetry.Exporter.OpenTelemetryProtocol.Grpc/OpenTelemetry.Exporter.OpenTelemetryProtocol.Grpc.csproj

```diff
@@ -5,6 +5,9 @@
     <Description>OpenTelemetry protocol exporter over gRPC for OpenTelemetry .NET</Description>
     <PackageTags>$(PackageTags);OTLP;gRPC</PackageTags>
     <MinVerTagPrefix>core-</MinVerTagPrefix>
+    <Nullable>enable</Nullable>
```

Comment:
Would it be better if we have these on-by-default in the common project template, and opt-out in existing projects which don't meet the bar?